### PR TITLE
net-misc/modemmanager: 1.14.8 version bump

### PR DESCRIPTION
**Pull request by candrews**
* net-libs/libmbim: 1.24.4 version bump
* net-libs/libqmi: 1.26.6 version bump
* net-misc/modemmanager: 1.14.8 version bump

### CODE REVIEW

File: net-misc/modemmanager/modemmanager-1.14.8.ebuild [hunk 1]
```diff
@@ -0,0 +1,130 @@
+# Copyright 1999-2021 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=6
```
Comment (dm0-) [inline net-misc/modemmanager/modemmanager-1.14.8.ebuild:4]: There is also an EAPI 7 update for this in #18813.

Comment (candrews) [reply net-misc/modemmanager/modemmanager-1.14.8.ebuild:4]: Thanks! I've incorporated the changes in https://github.com/gentoo/gentoo/pull/18813/commits/b941b20490fe11a27e41b6d5dfd504bd6767cf6b

**Comment by mattst88** [comment]
I fixed a handful of things here: dropping IUSE=static-libs, disabling -Werror, etc.

**Comment by candrews** [comment]
Thank you!

**Pull request closed without merging**